src/test/java/SpeechRankGUITests/src/main/java/configuration/LocalWebDriverProperties.java

```diff
@@ -0,0 +1,24 @@
+package configuration;
+
+import driver.manager.BrowserType;
+
+public class LocalWebDriverProperties {
+
+
+
+    public static BrowserType getLocalBrowser() {
```

Comment:
I'd suggest to move all the methods from `LocalWebDriverProperties`, `AppProperties`, `LocalWebDriverProperties` and `TestRunProperties` into `ConfigurationProperties`